test/commands/apextests/list.test.ts

```diff
@@ -3,7 +3,7 @@ import { expect } from 'chai';
 import { stubSfCommandUx } from '@salesforce/sf-plugins-core';
 import ApextestsList from '../../../src/commands/apextests/list.js';
 
-const TEST_LIST = ['SampleTest', 'SuperSampleTest', 'Sample2Test', 'SuperSample2Test', 'SampleTriggerTest'];
+const TEST_LIST = ['Sample2Test', 'SampleTest', 'SampleTriggerTest', 'SuperSample2Test', 'SuperSampleTest'].sort();
```

Comment:
**suggestion:** you could sort it yourself :)

Reply:
Sure, my but my rationale here is to facilitate when/if in the future we add more classes/samples, so we don't need to sort them manually when adding to the test.